discord/interactions.py

```diff
@@ -163,6 +163,7 @@ def _from_data(self, data: InteractionPayload):
         self.locale: Optional[str] = data.get("locale")
         self.guild_locale: Optional[str] = data.get("guild_locale")
         self.custom_id: Optional[str] = self.data.get("custom_id") if self.data is not None else None
+        self._app_permissions: int = int(data.get("app_permissions"))
```

Comment:
BTW, `app_permissions` is technically optional so you should give a default value to the `get` function (the default value would be 0)

Reply:
it's not optional in the api.
it's either the apps perms or everyone perms or 0.